src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -953,6 +972,24 @@ public override ImmutableArray<SyntaxReference> DeclaringSyntaxReferences
             }
         }
 
+        internal sealed override DeclarationScope Scope
+        {
+            get
+            {
+                DeclarationScope value;
+                if (!_packedFlags.TryGetScope(out value))
+                {
+                    // PROTOTYPE: Should we silently drop [LifetimeAnnotation] values that are invalid
```

Comment:
Is the intent of the use-site error to prevent calling a method with annotations that might be meaningful but this version of the compiler does not understand?

Reply:
In that vein, it might even be good to reject usage of LifetimeAnnotation when the constructor signature doesn't match something we recognize.